pkg/kubelet/volumemanager/reconciler/reconciler_test.go

```diff
@@ -2389,7 +2389,7 @@ func TestReconcileWithUpdateReconstructedFromAPIServer(t *testing.T) {
 	assert.Empty(t, reconciler.volumesNeedUpdateFromNodeStatus)
 
 	attachedVolumes := asw.GetAttachedVolumes()
-	assert.Equalf(t, len(attachedVolumes), 2, "two volumes in ASW expected")
+	assert.Lenf(t, attachedVolumes, 2, "two volumes in ASW expected")
```

Comment:
Why `Lenf`?  There are no printf args.  Can you followup or add a commit here to change this and any other cases ?

Reply:
It's the opinionated way of Anton.

As long there is a text, he wants to use the `f` method, even if there are no args.

I disliked it a lot at first. They I considered the implementation. It requires to use a variadic to use one optional text.

So even if I find more logical to use the `f` form, when there are sprintf like args. I now understand his point.

Reply:
ok